core/src/main/kotlin/ml/GeneticTesting.kt

```diff
@@ -0,0 +1,147 @@
+package ml
+
+import core.*
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
+import java.io.File
+import java.lang.Exception
+import java.util.logging.Level
+import java.util.logging.Logger
+import kotlin.math.roundToInt
+import kotlin.random.Random
+
+class GeneticTesting {
```

Comment:
I think, it should be moved to `test` submodule OR it should be renamed if it's not a test actually

Reply:
Fixed